gensim/sklearn_integration/sklearn_wrapper_gensim_ldamodel.py

```diff
@@ -25,13 +25,12 @@ class SklLdaModel(BaseSklearnWrapper, TransformerMixin, BaseEstimator):
     """
 
     def __init__(
-            self, num_topics=100, id2word=None,
-            chunksize=2000, passes=1, update_every=1,
-            alpha='symmetric', eta=None, decay=0.5, offset=1.0,
-            eval_every=10, iterations=50, gamma_threshold=0.001,
-            minimum_probability=0.01, random_state=None):
+            self, num_topics=100, id2word=None, chunksize=2000, passes=1,
+            update_every=1, alpha='symmetric', eta=None, decay=0.5,
+            offset=1.0, eval_every=10, iterations=50, gamma_threshold=0.001,
+            minimum_probability=0.01, random_state=None, scorer='c_v'):
         """
-        Sklearn wrapper for LDA model. derived class for gensim.model.LdaModel .
+        Sklearn wrapper for LDA model.
```

Comment:
@menshikh-iv I removed the comment "_derived class for gensim.model.LdaModel_" because the class `SklLdaModel` is no longer a derived class of `LdaModel` class (we are now using composite design pattern).